django_typeform/forms.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8
+"""Define TypeformMixin."""
+import hashlib
+
+from django.template.loader import render_to_string
+from django.utils.crypto import get_random_string
+
+from .responses import TypeformResponses
+
+
+class TypeformMixin(object):
+    """
+    Extend a Form class to add Typeform funcitonality.
+    """
+    typeform_id = None
+    typeform_url = None
+    typeform_template = 'django_typeform/typeform.html'
+
+    def __init__(self, *args, **kwargs):
+        self.typeform_url = kwargs.pop('typeform_url', self.typeform_url)
+        if self.typeform_url:
+            if self.typeform_url[-1] == '/':						# Skip trailing /
```

Comment:
There's some tabs happening here between then `:` and the `#` was this intentional?

Reply:
Unintentional...